Damiencastro/Midas-App
# Feature/password expiration check

Add password expiration check and change password functionality

Changes made:
- Added PasswordExpirationService for checking password age
- Added ChangePasswordComponent for password updates
- Modified login flow to check password expiration
- Added system notifications for password status

Required changes:
- Requires Firestore schema update to add passwordLastChanged field to user documents

Testing needed:
- Verify password expiration check on login
- Test password change workflow
- Confirm notifications are working
- Set up Firestore schema update
